docs/resources/lakehouse_monitor.md

```diff
@@ -0,0 +1,135 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_lakehouse_monitor Resource
+
+This resource allows you to manage [Lakehouse Monitors](https://docs.databricks.com/en/lakehouse-monitoring/index.html) in Databricks. 
+
+A `databricks_lakehouse_monitor` is attached to a [databricks_sql_table](sql_table.md) and can be of type timeseries, snapshot or inference. 
+
+## Example Usage
+
+```hcl
+
+resource "databricks_catalog" "sandbox" {
+  name    = "sandbox"
+  comment = "this catalog is managed by terraform"
+  properties = {
+    purpose = "testing"
+  }
+}
+
+resource "databricks_schema" "things" {
+    catalog_name = databricks_catalog.sandbox.id
+    name         = "things"
+    comment      = "this database is managed by terraform"
+    properties = {
+      kind = "various"
+    }
+  }
+  
+resource "databricks_table" "myTestTable" {
+    catalog_name = "main"
+    schema_name = databricks_schema.things.name
+    name = "bar"
+    table_type = "MANAGED"
+    data_source_format = "DELTA"
+
+    column {
+        name      = "timestamp"
+        position  = 1
+        type_name = "INT"
+        type_text = "int"
+        type_json = "{\"name\":\"id\",\"type\":\"integer\",\"nullable\":true,\"metadata\":{}}"
+    }
+}
+
+resource "databricks_lakehouse_monitor" "testTimeseriesMonitor" {
+    table_name = "${databricks_catalog.sandbox.name}.${databricks_schema.things.name}.${databricks_table.myTestTable.name}"
```

Comment:
Hmm I wonder if this is a bit too much, we ended up separating this in a UC Model into 3 parameters for this reason: https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/registered_model

Reply:
According to their python api docs (https://api-docs.databricks.com/python/lakehouse-monitoring/latest/databricks.lakehouse_monitoring.html#databricks.lakehouse_monitoring.create_monitor) , it can be `{catalog}.{schema}.{table}` or `{schema}.{table}` or `{table}` and the api fills in the current catalog or schema. So we technically dont need to split it up and force users to fill in all three fields. I think this should be the recommendation to fill in all three, but if the user reads other docs, they should be able to just use all three of the options

Reply:
Got it, so if someone doesn't specify schema and catalog via Terraform though, what would be the "current" catalog and schema it would infer?

Reply:
No, in for `databricks_sql_table` you always specify catalog & schema and `id` will be three-level name